README.md

```diff
@@ -208,14 +208,30 @@ Mac: Install autotools using [MacPorts](https://www.macports.org/install.php)
 port install automake autoconf libtool pkgconfig
 ```
 
-Windows: Install autotools using [MSYS2](http://www.msys2.org/)
+Windows: The [MSYS2](http://www.msys2.org/) does not support Node.js anymore. We have to find another way to build it using the Windows platform.
+
+The old (obsolete) instructions:
+
+Install autotools using [MSYS2](http://www.msys2.org/)
 ```bash
 pacman -S make autoconf automake libtool pkgconfig
```

Comment:
These instructions don't work on newer macOS
```
nadehz@Adams-MacBook-Pro-5 opus-recorder % port install automake autoconf libtool pkgconfig

Error: Current platform "darwin 21" does not match expected platform "darwin 19"
Error: If you upgraded your OS, please follow the migration instructions: https://trac.macports.org/wiki/Migration
OS platform mismatch
    while executing
"mportinit ui_options global_options global_variations"
Error: /opt/local/bin/port: Failed to initialize MacPorts, OS platform mismatch
```

Reply:
I believe that's a problem of the previous installation of the [macports](https://www.macports.org/install.php) on the macOS before it being upgraded. So you have to either reinstall macports or perform the [migration procedure](https://trac.macports.org/wiki/Migration)